preview/MsixCore/msixmgr/msixmgr.cpp

```diff
@@ -105,21 +105,34 @@ int main(int argc, char * argv[])
 
             shared_ptr<IInstalledPackage> packageInfo;
             HRESULT hr = packageManager->FindPackage(cli.GetPackageFullName(), packageInfo);
-            if (packageInfo == NULL || FAILED(hr))
+
+            if (packageInfo == NULL)
             {
-                std::wcout << std::endl;
-                std::wcout << L"No packages found " << hr << std::endl;
-                std::wcout << std::endl;
+                hr = packageManager->FindPackageByFamilyName(cli.GetPackageFullName(), packageInfo);
+                if (FAILED(hr))
+                {
+                    std::wcout << L"Failed to determine findpackage results" << hr << std::endl;
+                }
+
+                if (packageInfo == NULL)

```

Comment:
>if [](start = 16, length = 2)

else if

Reply:
I don't think I can make this else if since package info might be null initially if not found by fullname, but might be populated by the time FindPackageByFamilyName method runs. If 'else if' is introduced, this portion will not execute and results will not be printed even when found from the FindPackageByFamilyName method.

Reply:
this else if would be the else if of the FAILED(hr) above. packageInfo should be null if it fails, and you wouldn't want to log twice.

Reply:
Yes, updated that.